index.js

```diff
@@ -65,8 +65,9 @@ module.exports = function (options) {
 
 function toLink(pkg, num, words) {
   var res = '';
-  res += link(pkg.name, pkg.homepage);
-  res += truncate(pkg.description, pkg.homepage, words);
+  var homepage = pkg.homepage.split('#readme').join('')
```

Comment:
would you mind changing this line to:

```js
var homepage = pkg.homepage.replace(/#readme$/, '');
```
 just to be safe, since homepage links with anchors are perfectly valid, and it's possible (maybe unlikely, but still...) that a user might intentionally define an anchor that begins with `#readme`.